zeebe/expression-language/src/test/java/io/camunda/zeebe/el/EvaluationResultTest.java

```diff
@@ -227,6 +232,34 @@ public void contextWithDateExpression() {
     assertThat(evaluationResult.toBuffer()).isEqualTo(asMsgPack("{'x':\"2020-04-02\"}"));
   }
 
+  @Test
+  public void shouldReturnNormalizedDuration() {
+    final var evaluationResult = evaluateExpression("=duration(\"P5D\")");
```

Comment:
🔧 We could show the normalization by starting with a non-normalized format:

- `PT120H` -> `P5D`